classes/PaymentModule.php

```diff
@@ -664,6 +664,7 @@ public function validateOrder(
                                 '{total_shipping_tax_incl}' => Tools::getContextLocale($this->context)->formatPrice($order->total_shipping_tax_incl, $this->context->currency->iso_code),
                                 '{total_wrapping}' => Tools::getContextLocale($this->context)->formatPrice($order->total_wrapping, $this->context->currency->iso_code),
                                 '{total_tax_paid}' => Tools::getContextLocale($this->context)->formatPrice(($order->total_paid_tax_incl - $order->total_paid_tax_excl), $this->context->currency->iso_code),
+                                '{recycled_packaging_label}' => $order->recyclable ? $this->trans('Yes', [], 'Shop.Theme.Global') : $this->trans('No', [], 'Shop.Theme.Global'),
```

Comment:
Instead of passing the label, why not passing the boolean so you're free to choose what you want to display in the template?

Reply:
@atomiix we can't use condition in e-mails templates. (it's a very good improvement but actually we can just change {toto} by Toto We can't do if toto )

The only way to use condition is inside : 

https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/MailTemplate/MailTemplateTwigRenderer.php#L122-L128

And it's only for generate template

Reply:
You're right! I forgot about that 😄